clusterloader2/testing/load/deployment.yaml

```diff
@@ -51,6 +67,11 @@ spec:
         - name: configmap
           configMap:
             name: {{.BaseName}}-{{.Index}}
+{{if .ENABLE_DNSTESTS}}
```

Comment:
If we apply my above comment in configmap.yaml, then we can do this also unconditionally.

[In other words - we should probably split this update and configmap update to a sepraate PR (and done the unconditionally) and leave the rest in this PR.]
@mborsz - thoughts?

Reply:
See answer above.

Reply:
As discussed @wojtek-t . I think it makes sense to simply always mount all keys and using one of them, so let's remove this "items" section from here.

The config should be available under /var/configmap/all-queries anyway

Reply:
sounds good.. thanks for submitting the other PR!

Reply:
Let's remove it - you no longer need it.

We discussed that and we belive it makes sense to always mount everything - it will be even simpler.